chat-plugins/roomsettings.js

```diff
@@ -399,6 +408,36 @@ exports.commands = {
 	},
 	capsfilterhelp: ["/capsfilter [on/off] - Toggles filtering messages in the room for EXCESSIVE CAPS. Requires # & ~"],
 
+    emojis: 'emojifilter',
+    emoji: 'emojifilter',
+    emojifilter : function (target, room, user) {
+		if (!target) {
+			const emojiSetting = (room.filterEmojis ? "ON" : "OFF");
+			return this.sendReply(`This room's emoji filter is currently: ${emojiSetting}`);
+		}
+		if (!this.canTalk()) return;
+		if (!this.can('editroom', null, room)) return false;
+
+		if (target === 'enable' || target === 'on' || target === 'true') {
+			if (room.filterEmojis) return this.errorReply(`This room's emoji filter is already ON`);
+			room.filterEmojis = true;
+		} else if (target === 'disable' || target === 'off' || target === 'false') {
+			if (!room.filterEmojis) return this.errorReply(`This room's emoji filter is already OFF`);
+			room.filterEmojis = false;
+		} else {
+			return this.parse("/help emojifilter");
+		}
+		const emojiSetting = (room.filterEmojis ? "ON" : "OFF");
+		this.privateModCommand(`(${user.name} turned the emoji filter ${emojiSetting})`);
+
+		if (room.chatRoomData) {
+			room.chatRoomData.filterEmojis = room.filterEmojis;
+			Rooms.global.writeChatRoomData();
+		}
+    },
+
+    emojifilterhelp: ["/emojifilter [on/off] - Toggles filtering messages in the room for emojis. Requires # & ~"],
```

Comment:
I'm gonna change this in my next commit. Didn't notice when I started because of vim config.